src/logistration/RegistrationFailure.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+import Alert from './Alert';
+
+const RegistrationFailureMessage = (props) => {
+  const errorMessage = props.errors;
+  const userErrors = [];
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
+  }, [errorMessage]);
+
+  Object.keys(errorMessage).forEach((key) => {
+    const errors = errorMessage[key];
+    const errorList = errors.map((error) => {
+      const errorMsg = error.user_message;
+      return (
+        <li key={error} style={{ textAlign: 'left' }}>
+          <FormattedMessage
+            id="logistration.registration.request.failure.header."
+            defaultMessage="{errorMsg}"
```

Comment:
During the extraction of strings this message won't be available, so formatting is unnecessary here.

Reply:
As the error messages would be backend generated and translate its only acting as a container.

Reply:
As the error messages would be backend generated and translate its only acting as a container.

Reply:
I still don't understand the reason for it, why can't we simply render it?

Reply:
Ill see that.

Reply:
Done